mkt/api/middleware.py

```diff
@@ -262,17 +262,27 @@ def process_response(self, request, response):
             getattr(request, 'CORS_HEADERS',
                     ('X-HTTP-Method-Override', 'Content-Type')))
 
-        if fireplacey or getattr(request, 'CORS', None):
+        error_allowed_methods = []
+        # If settings.DEBUG is True, these skip the 404 and 500 handlers
```

Comment:
They are and that's seems a valid point and would reduce the lines of code, I'll make that change.